content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
+If the issue is not reproduced: **It’s a regression!** Test all the latest patch versions of the current minor version (e.g. 1760, 1761, 1762...) to find out in which version the issue has been introduced. If motivated, look for the PR in question. Then the issue follows the normal course: go to the next step. In addition, it has to be posted on #qa-core channel (PS Slack) and #dev-core (Public Slack). The Project must then be labeled by selecting the **label corresponding to the version in which the regression was detected.**
 
 {{% notice note %}}
-Regressions for the **last minor or major stable version** must be placed in the Maxi Kanban, in the first column. 
+The regression of the last minor or major stable version must be placed in the [patch backlog](https://github.com/PrestaShop/PrestaShop/projects/20) kanban, in the first column.
+{{% /notice %}}
 
-Regression for the **develop branch** must be placed in the Kanban of the corresponding next minor or major version.
+{{% notice note %}}
+The regression of the develop branch must be placed in the Kanban of the corresponding next minor or major version.
 {{% /notice %}}
 
-#### Labelling
+### Labelling
 
-**This step is crucial.** Labelling consists on tagging the issue according to its different characteristics. This step is **critical** because it helps us find a bug when we’re looking for a duplicate. The different tags are:
+**This step is critical** cause **labelling** consists on tagging the issue according to its different characteristics and it helps us find a bug when we’re looking for a `duplicate`. The different tags are:
 
-- Type (bug or feature),
-- [Severity (major, minor, ...)][how-issues-are-sorted],
-- Regression or not,
-- Verified (if we reproduce the issue)
-- Category (FO/BO + affected page or concerned feature),
+- Type (`Bug` or `Task`),
+- [Severity (major, minor, ...)](../../../../get-involved/report-issues/how-issues-are-sorted/)
+- *Deprecated - Regression or not (since August 2023),*
+- `Verified` (if we reproduce the issue)
+- Category (`FO`/`BO`),
+- Affected page or concerned feature (ex: `Order`, `Module`, …),
 - Affected version(s),
 - Status (NMI, Needs Specs, Ready...)
 - ... (non-exhaustive list)
 
-A list of labels and their use is [available here](https://github.com/PrestaShop/PrestaShop/labels).
+A list of labels and their use is [available here](https://docs.google.com/spreadsheets/d/118iqWIkW24Rwn6eY8_7FnPhAacGXwA2RdGPzR_LXvy4/edit#gid=0).
```

Comment:
@Progi1984 not available publicly

Reply:
Yes, it will be replaced in the next update (all internals documents will be published) by an another member of the QA Council.